play/apps/play/models.py

```diff
@@ -0,0 +1,41 @@
+from apps.play.fields import ShortUUIDField
+from django.db import models
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.base_user import AbstractBaseUser
+
+
+class User(AbstractBaseUser, PermissionsMixin):
+    id = ShortUUIDField(prefix="user", max_length=128, primary_key=True)
+    team_id = models.CharField(max_length=128)
+
+    # is_admin = models.BooleanField(default=False)
+    is_bounty = models.BooleanField(default=False)
```

Comment:
what do these 3 fields mean?

Reply:
my intent was to flag the different user models as:
 - bounty snake user: ones who can initialize custom bounty snake games
 - registration desk users: those who are able to register/create new users/teams
 - tournament runners: those who are able to manage the tournament games, i.e. the person running the tournament tree, any helpers, and the casters.
I believe these don't need to be in on this p.r. as the below comment suggests. Should I remove them?

Reply:
Can we turn this into some sort of Role field, i.e. a user can have multiple roles?